recipes/geeadd/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "geeadd" %}
+{% set version = "0.4.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4ff2312cf7304c38f5d16daa503c148b898ddb14a7f04f746fe56b61f7d612f6
+
+build:
+  number: 0
+  entry_points:
+    - geeadd=geeadd.geeadd:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools >=19.6
+  run:
+    - beautifulsoup4 >=4.9.0
+    - earthengine-api >=0.1.217
+    - gitpython >=3.1.1
```

Comment:
I went through https://github.com/samapriya/gee_asset_manager_addon/issues/10
Is gitpython still a valid dependency? I don't see it it with the Requires-Dist field in the METADATA file inside the wheel uploaded on PyPI.

Reply:
removed gitpython